src/vendors/CredentialCard.js

```diff
@@ -29,7 +29,7 @@ export default function CredentialCard(props) {
     }, [])
 
     function color() {
-        Math.seedrandom(credentialType)
+        //Math.seedrandom(credentialType)
```

Comment:
Why you comment out this line ?

Reply:
earlier it was giving an error like Math.seedrand() is not function.but it is working now I will change it .